share/revng/test/tests/analysis/Decompilation/clang/dual-case-switch.c

```diff
@@ -0,0 +1,35 @@
+/*
+ * This file is distributed under the MIT License. See LICENSE.md for details.
+ */
+#include <stdio.h>
+
+int f(int arg1) {
+  int return_value;
+  while (arg1 > 0) {
+    if (arg1 == 1) {
+      return_value = 10;
+      printf("Argument equal to 1");
+      goto LABEL_EXIT_1;
+    } else if (arg1 == 2) {
+      return_value = 20;
+      printf("Argument equal to 2");
+      goto LABEL_EXIT_2;
+    }
+  }
+
+LABEL_EXIT_1:
+  printf("Exit 1");
+  goto LABEL_EXIT_COMMON;
+
+LABEL_EXIT_2:
+  printf("Exit 2");
+
+LABEL_EXIT_COMMON:
+  return return_value;
+}
+
+int main() {
+  int a = 0;
```

Comment:
Is this needed?

Reply:
Nope, it was a leftover from a previous implementation. Removed this in https://github.com/revng/revng-qa/commit/7d9d32d70292d147d1eec204ac75342c74c3fd68